frontend/public/components/monitoring/query-browser.tsx

```diff
@@ -746,17 +744,18 @@ const QueryBrowser_: React.FC<QueryBrowserProps> = ({
 
   if (isRangeVector) {
     return (
-      <GraphEmptyState title="Ungraphable results">
-        Query results include range vectors, which cannot be graphed. Try adding a function to
-        transform the data.
+      <GraphEmptyState title={t('monitoring~Ungraphable results')}>
+        {t(
+          'monitoring~Query results include range vectors, which cannot be graphed. Try adding a function to transform the data.',
+        )}
       </GraphEmptyState>
     );
   }
 
   if (isDatasetTooBig) {
     return (
-      <GraphEmptyState title="Ungraphable results">
-        The resulting dataset is too large to graph.
+      <GraphEmptyState title={t('monitoring~Ungraphable results')}>
```

Comment:
We're no longer using the `monitoring` namespace, so these should be `t('public~...)`

Reply:
Removed the `monitoring` namespace. Put the English strings into public.json.